pom.xml

```diff
@@ -81,7 +81,7 @@
     <seam.version>2.3.1.Final</seam.version>
     <slf4j.version>1.7.12</slf4j.version>
     <gwteventservice.version>1.2.1</gwteventservice.version>
-    <okapi.version>0.22</okapi.version>
+    <okapi.version>0.29-SNAPSHOT</okapi.version>
```

Comment:
Okapi 0.26 doesn't include jericho 3.4-dev. Any chance we could use that instead for qt-ts support, until we can sort out a properly tagged version of jericho 3.4-dev?